Accessibility/AriaNotify/explainer.md

```diff
@@ -366,67 +353,118 @@ if (percent < 100) {
-`<iframe>`, such as `allow-notifications`.
-
-## Relationship to ARIA Live Regions
-There are some similarities between `ariaNotify` and the existing ARIA live regions. This section maps the existing ARIA
-live region configuration attributes to the options available with `ariaNotify`: 
- - `aria-live="assertive"` is the equivalent of `"priority: important"` and `"interrupt: none"`
- - `aria-live="polite"` is the equivalent of `"priority: none"` and `"interrupt: none"`
-
-Beyond the above, the additional functionality provided by `ariaNotify` is not supported and cannot be mapped back
-directly to ARIA live regions. 
-
-## Fallback 
-In the case of browsers that do not yet support `ariaNotify`, we propose the following fallback mechanism using the same
-backend as the existing ARIA live regions: 
- - The message payload for `ariaNotify` is equivalent to the contents of an ARIA live region. 
- - The `notificationId` is dropped entirely. 
- - `"priority: important"` and `"priority: none"` correspond to `aria-live="assertive"` and `aria-live="polite"` ARIA
- live attributes, respectively. 
- - ARIA live regions do not support interruptibility, so all behavior of `interrupt` defaults to `none`.  
+##### Fallback
+In the case of browsers that do not yet support `ariaNotify`, ARIA live regions do not support interruptibility, so all behavior of `interrupt` defaults to `none`. 
 
-Note that there is no exact mapping of `ariaNotify` back to ARIA live regions, and our proposal reflects a best effort
+The addition of `interrupt` would mean that there is no exact mapping of `ariaNotify` back to ARIA live regions, and our proposal reflects a best effort
 to achieve similar behavior. There are cases where we will not be able to get the intended behavior using ARIA live
 regions:
 
-#### Example 6 
-```
+```js
 element.ariaNotify("This message is normal.",
-    { "priority": "none",
+    { "priority": "normal",
       "interrupt": "none"}); 
 
 element.ariaNotify("This message should interrupt",
-    { "priority": "none", "interrupt": "all" }); 
+    { "priority": "normal", "interrupt": "all" }); 
 ```
 
 In the above case, when `ariaNotify` is supported, the expected behavior would be for the second notification to silence
-the current one and flush all other queued notifications from element with priority: `"none"`. However, the fallback is
+the current one and flush all other queued notifications from element with priority: `"normal"`. However, the fallback is
 not able to silence or flush existing notifications, as that behavior is not supported in ARIA live regions.  In the
 case that the web browser does not yet support `ariaNotify`, it is the responsibility of the web author to detect and
 fallback to ARIA live regions.  The above conversion may serve as a guide on how to do so. One can detect whether or not
 `ariaNotify` is supported by checking if the method exists on the document or element in question: 
 
-```
+```js
```

Comment:
Does this need to handle three cases now? No ariaNotify, ariaNotify without interrupt, and ariaNotify with interrupt.

Reply:
Yeah actually that is probably a good point. Although for browsers that don't support it, it wouldn't probably parse anyways.

I think if we do add `interrupt` later, we will need to add some way to do feature detection to the `ariaNotify` API itself. Which reminds me, Alex had pointed me to https://developer.mozilla.org/en-US/docs/Web/API/ClipboardItem/supports_static which is something we should potentially consider adding to our proposal for `interrupt` to provide authors a way to detect if a browser supports that property yet